service/worker/deletenamespace/reclaimresources/activities.go

```diff
@@ -85,12 +81,7 @@ func NewLocalActivities(
 }
 
 func (a *LocalActivities) IsAdvancedVisibilityActivity(_ context.Context, nsName namespace.Name) (bool, error) {
-	switch a.visibilityManager.GetReadStoreName(nsName) {
-	case elasticsearch.PersistenceName, mysql.PluginName, postgresql.PluginName, postgresql.PluginNamePGX, sqlite.PluginName:
-		return true, nil
-	default:
-		return false, nil
-	}
+	return true, nil
 }
 
 func (a *LocalActivities) CountExecutionsAdvVisibilityActivity(ctx context.Context, nsID namespace.ID, nsName namespace.Name) (int64, error) {
```

Comment:
```suggestion
func (a *LocalActivities) CountExecutionsActivity(ctx context.Context, nsID namespace.ID, nsName namespace.Name) (int64, error) {
```
And create an alias for `CountExecutionsAdvVisibilityActivity` and todo to remove it after 1.26.

Reply:
Same for `EnsureNoExecutionsAdvVisibilityActivity`.

Reply:
Will create task to track this leftover.